docs/_config.yaml

```diff
@@ -1,5 +1 @@
-future: true
-encoding: "UTF-8"
-include:
-  - "_*_.html"
-  - "_*_.*.html"
+future: true\nencoding: "UTF-8"\ninclude:\n  - "_*_.html"\n  - "_*_.*.html"
```

Comment:
This doesn’t look right...

Reply:
```makefile
typedoc:
	@rm -rf docs
	@typedoc --ignoreCompilerErrors --out ./docs --mode modules --includeDeclarations --excludeExternals --name opine ./src
	@make fmt
	@make fmt
	@echo 'future: true\nencoding: "UTF-8"\ninclude:\n  - "_*_.html"\n  - "_*_.*.html"' > ./docs/_config.yaml

```
hmmm i see, there is a line in the makefile where you write that line into doc/_config.yaml file, see above,
unless this works diferently on windows

Reply:
but i see what you trying to do, i am going to update that manually

Reply:
Ta - It’s clearly not having the desired effect on windows, works on MacOS / Linux. Unfortunately don’t have a windows PC to test with!

Reply:
@asos-craigmorten ok Fixed 🙂